ydb/services/ydb/ydb_ldap_login_ut.cpp

```diff
@@ -99,9 +99,9 @@ void InitLdapSettingsWithEmptyBindPassword(NKikimrProto::TLdapAuthentication* ld
 
 class TLoginClientConnection {
 public:
-    TLoginClientConnection(std::function<void(NKikimrProto::TLdapAuthentication*, ui16, TTempFileHandle&)> initLdapSettings)
+    TLoginClientConnection(std::function<void(NKikimrProto::TLdapAuthentication*, ui16, TTempFileHandle&)> initLdapSettings, bool isBuiltinAuthMechanismDisabled = true)
```

Comment:
I wouldn't recommend doing it with bools in the constructor, but it's a UT so it's not that important